docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2342/2342_6.md

```diff
@@ -0,0 +1,21 @@
+# Check restricted only to specific users groups "system" parameters via CLI
+
+Test verifies that 
+- If the user launches a job with a parameter, which is not allowed - a meaningful error shall be thrown and job shall NOT be started
+
+
+**Prerequisites**:
+- User group (*User_group*)
+- Non-admin user isn't added to the *User_group* (*User1*)
+- Perform [2342_1](2342_1.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin *User1* from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select the test tool | |
+| 4 | Launch a tool with default settings | |
+| 5 | At the **Runs** page, click the just-launched run | | 
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the command: `pipe run -di <docker image> --CP_FSBROWSER_ENABLED true` | <li> job isn't started <li>  The command output is `An error has occurred while starting a job: "CP_FSBROWSER_ENABLED" parameter is not permitted for use`|
```

Comment:
Add the block "After", in which `CP_FSBROWSER_ENABLED` parameter from the **launch.system.parameters** system preference should be returned to the view as it was before the [2342_1](2342_1.md) case